ui/components/app/modals/transaction-already-confirmed/__snapshots__/transaction-already-confirmed.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Transaction Already Confirmed modal should match snapshot 1`] = `<div />`;
```

Comment:
This snapshot is empty as it's a modal so floating above the actual component.

I think we can instead check `baseElement` rather than `container` to include the modal?

Reply:
Fixed in https://github.com/MetaMask/metamask-extension/pull/22943/commits/7c64152b15f6224a390e6f8200288f56fefc0011